lib/nsxv.py

```diff
@@ -503,30 +505,83 @@ def _TranslatePolicy(self, pol, exp_info):
           term = self.FixHighPorts(term, af=af)
           if not term:
             continue
-          new_terms.append(Term(term, filter_type, 4))
+          new_terms.append(Term(term, filter_type, applied_to, 4))
 
         if filter_type == 'inet6':
           af = 'inet6'
           term = self.FixHighPorts(term, af=af)
           if not term:
             continue
-          new_terms.append(Term(term, filter_type, 6))
+          new_terms.append(Term(term, filter_type, applied_to, 6))
 
         if filter_type == 'mixed':
           if 'icmpv6' not in term.protocol:
             inet_term = self.FixHighPorts(term, 'inet')
             if not inet_term:
               continue
-            new_terms.append(Term(inet_term, filter_type, 4))
+            new_terms.append(Term(inet_term, filter_type, applied_to, 4))
           else:
             inet6_term = self.FixHighPorts(term, 'inet6')
             if not inet6_term:
               continue
-            new_terms.append(Term(inet6_term, filter_type, 6))
+            new_terms.append(Term(inet6_term, filter_type, applied_to, 6))
 
-      self.nsxv_policies.append((header, filter_name, [filter_type],
+      self.nsxv_policies.append((header, filter_name, [filter_type], section_id,
```

Comment:
This can be part of the header and in fact is already.
(Pdb) print header.FilterOptions('nsxv')
[u'inet', u'1010', u'securitygroup', u'securitygroup-Id']